fe1-web/src/features/rollCall/components/RollCallOpen.tsx

```diff
@@ -52,6 +52,15 @@ const RollCallOpen = ({
 
   const [popToken, setPopToken] = useState('');
   const [hasWalletBeenInitialized, setHasWalletBeenInitialized] = useState(hasSeed());
+  const allAttendees = useMemo(() => {
+    if (isOrganizer && popToken !== '') {
+      if (scannedPopTokens && scannedPopTokens.length > 0) {
+        return scannedPopTokens;
+      }
+      return [PublicKey.fromState(popToken)];
+    }
+    return scannedPopTokens || [];
+  }, [isOrganizer, popToken, scannedPopTokens]);
```

Comment:
I would at least add a comment that you want to add the organizer's key if it is not already present. Otherwise this looks a bit confusing as you implicitly assume that the organizer key is always present if `scannedPopTokens.length > 0` (which is okay but might no be clear without any previous knowledge)